css/styles.css

```diff
@@ -14,15 +14,15 @@
 }
 
 body {
-  font-family: 'Lato', sans-serif;
-  font-size: 13px;
+  font-family: 'open sans - bold', sans-serif;
```

Comment:
I don't know if this will work without importing the font? or I could be totally wrong...